bokehjs/src/coffee/api/plugin.coffee

```diff
@@ -0,0 +1,27 @@
+
+define [
+  "underscore"
+  "jquery"
+  "common/logging"
+  "./figure"
+  "./helpers"
+], (_, $, Logging, figure, helpers) ->
+
+  logger = Logging.logger
+  show = helpers.show
+
+  _api = {
+    "figure": figure
+  }
+
+  $.fn.bokeh = (type, args) ->
```

Comment:
@mattpap I see what you are saying, however it is that case that all the examples do work using `$().bokeh)`, regardless of whether they use `require` or are a bare script. I am not sure why that should be the case, though, given what you have mentioned. Are there any immediate changes you would propose?